tests/test_difference.py

```diff
@@ -0,0 +1,38 @@
+import torch
```

Comment:
can you also do the following checks:

- number of points sampled in case point is divisible by the number of geometries
- number of points sampled in case point is **not** divisible by the number of geometries
- test also difference of 3 geometries